### PR TITLE
Add CONTRIBUTING file to the project and ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by mere-teresa**
Please review and comment this PR.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+To contribute code:
+
+* [Set up the project](#project-setup).
+* Make any necessary changes to the source code.
+* Include any [additional documentation](#contribute-documentation) the changes might need.
+* Run tests: 
+  * unit tests: `npm run unit`
+  * e2e tests: `npm run e2e`
+  * all tests: `npm run test`
+* Let the Docker build pass (that verify that your contribution works as expected).
```
Comment (Amoki) [inline CONTRIBUTING.md]: Let the CircleCI build pass

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+We like code commits a lot! They're super handy, and they keep the project going and doing the work it needs to do to be useful to others.
+
+Code contributions of just about any size are acceptable.
+
+To contribute code:
+
+* [Set up the project](#project-setup).
+* Make any necessary changes to the source code.
+* Include any [additional documentation](#contribute-documentation) the changes might need.
+* Run tests:
```
Comment (Amoki) [inline CONTRIBUTING.md]: We don't have tests yet. But you can check if the docker still builds by using `docker build .`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  * all tests: `npm run test`
+* Let the Docker build pass (that verify that your contribution works as expected).
+* Write clear, concise commit message(s).
+* Dependency updates, additions, or removals must be in individual commits.
+* Go to https://github.com/bimdata/platform/pulls and open a new pull request with your changes.
+* If your PR is connected to an open issue, add a line in your PR's description that says `Fixes: #123`, where `#123` is the number of the issue you're fixing.
+
+Once you've filed the PR:
+
+* Maintainers will not review PRs until all checks pass (it means "until Docker builds").
```
Comment (Amoki) [inline CONTRIBUTING.md]: (it means "until CircleCI builds")

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+[Needs Collaborator](#join-the-project-team): Issue Tracker
+
+While anyone can comment on a PR, add feedback, etc, PRs are only *approved* by team members with Issue Tracker or higher permissions.
+
+PR reviews use [GitHub's own review feature](https://help.github.com/articles/about-pull-request-reviews/), which manages comments, approval, and review iteration.
+
+Some notes:
+
+* You may ask for minor changes ("nitpicks"), but consider whether they are really blockers to merging: try to err on the side of "approve, with comments".
+* *ALL PULL REQUESTS* should be let the Docker build pass.
```
Comment (Amoki) [inline CONTRIBUTING.md]: CircleCI

**Review by Amoki — CHANGES_REQUESTED** [review]

**Comment by mere-teresa** [comment]
I rewrote the CONTRIBUTING file to simplify it.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+If you have a question about this project, how to use it, or just need clarification about something:
+
+* Contact us: support@bimdata.io
+
+## Report an Issue
+
+If you find a security vulnerability, do NOT open an issue.
+E-mail us support@bimdata.io instead.
+
+If you run into an issue or bug 🐛 with the project, then open an Issue.
```
Comment (juliendubost) [inline CONTRIBUTING.md:47]: C'est quoi le caractère entre "bug" et "with" ?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+* One or more maintainers will use GitHub's review feature to review your PR.
+* If the maintainer asks for any changes, then edit your changes, push, and ask for another review.
+* If the maintainer decides to pass on your PR, they will thank you for the contribution and explain why they won't be accepting the changes. That's ok! We still really appreciate you taking the time to do it, and we don't take that lightly.
+* If your PR gets accepted, it will be marked as such, and merged into the `dev` branch soon after. Your contribution will be distributed to the masses as soon as the maintainer trigger a deployment process.
+
+# Contribute documentation
+
+Documentation is a super important, critical part of this project. Docs are how we keep track of what we're doing, how, and why. It's how we stay on the same page about our policies. And it's how we tell others everything they need in order to be able to use this project -- or contribute to it. So thank you in advance.
+
+Documentation contributions of any size are welcome! Feel free to file a PR even if you're just rewording a sentence to be more clear, or fixing a spelling mistake!
```
Comment (juliendubost) [inline CONTRIBUTING.md:127]: "to file" -> "to fill" ?

File: CONTRIBUTING.md (hunk 5 above)
Comment (Amoki) [reply CONTRIBUTING.md:47]: a "bug" emoji

File: CONTRIBUTING.md (hunk 6 above)
Comment (Amoki) [reply CONTRIBUTING.md:127]: to file is OK

File: CONTRIBUTING.md (hunk 5 above)
Comment (mere-teresa) [reply CONTRIBUTING.md:47]: Un émoji "bug" !  :bug:

**Review by Amoki — APPROVED** [review]

**Review by juliendubost — APPROVED** [review]

**Review by Amoki — APPROVED** [review]

**Pull request merged**